src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -63,12 +74,43 @@ public PesterCodeLensProvider(EditorSession editorSession)
                     new ClientCommand(
                         "PowerShell.RunPesterTests",
                         "Debug tests",
-                        new object[] { scriptFile.ClientFilePath, true /* Run in debugger */, pesterSymbol.TestName })),
+                        new object[] { scriptFile.ClientFilePath, true /* Run in debugger */, pesterSymbol.TestName, describeBlockLineNumber })),
             };
 
             return codeLensResults;
         }
 
+        /// <summary>
+        /// Used to determine the value of <see cref="_pesterV4_6_0_OrHigherAvailable"/> as a background task.
+        /// </summary>
+        private bool DeterminePesterVersion()
+        {
+            var powerShell = new PSCommand();
+            powerShell.AddCommand("Get-Module")
```

Comment:
Can omit the `; powerShell` here and chain the builder into the constructor if you like